src/easy_config/__init__.py

```diff
@@ -140,6 +140,7 @@ def load(
         *,
         _parse_files: bool = True,
         _parse_environment: bool = True,
+        _lookup_config_envvar: Optional[str] = None,
```

Comment:
Should this be configurable on a class level? What's really the difference if things are configured there or during the call to the function, which in theory only gets called once since this *should* be a singleton?

Reply:
just a thought. I'm happy with how this works

Reply:
Yeah I'm happy too.

Reply:
I mean, happy about this. `_lookup_config_envvar` doesn't make me happy in a general sense.